### DISCUSSION TITLE
Adding Portfolio section doesnt work

### DISCUSSION THREAD

**Discussion by minhaz725** [lxndrblz/anatole#495] [Q&A]
Tried this from wiki: "You can create an optional portfolio page that allows showcasing recent projects and publications. The entries within your portfolio are controlled by the portfolio.yml inside of your data folder. The portfolio.yml may look similar to the following structure:" I copied the yml that is inside example site/data and pasted it inside my root projects data folder. Nothing displays at portfolio tab (404). Can anyone help?

**Comment by yudistirasd** [comment]
you can try to create portfolio/_index.md in content folder.

**Reply by bigsolom** [reply]
This worked for me 👍
